.vscode/tasks.json

```diff
@@ -10,7 +10,10 @@
         "focus": false,
         "panel": "shared"
       },
-      "args": ["run", "bootstrap"],
+      "args": [
```

Comment:
Prettier results.

@xyc I was noticing that your prettier results were different from mine. Do you know if this still conflicts with the results you were seeing? I don't think my prettier has any custom setup. I'm using the out of the box extension 'esbenp.prettier-vscode'.

Reply:
I don't have custom setup either. Would it be the same results if you format it from command palette: "Format Document With..." then select prettier? I noticed that if I manually format it with "JSON Language Features" it produces the same result as your change here. In the "Format Document With..." you can also select "Configure Default Formatter" to use prettier as the default

Reply:
@xyc Thank you for pointing this out. I wasn't actually seeing that command palette option before. Reinstalling the plugin fixed the issue for me.